helpers/scrape.js

```diff
@@ -1,10 +1,11 @@
+// seems to work using import. good to resolve any remaining issues.
+// Please only include the needed helper code now - not sure what is still being used
```

Comment:
You can keep the runner files if they help you test - but maybe rename them to testScrapeRunner.js and testSyncRunner.js.